testing/tests/DevExpress.ui.widgets.pivotGrid/sortable.tests.js

```diff
@@ -1195,9 +1214,11 @@ QUnit.module('sortable when source item is hidden', {
         $('<div id=\'first-group\' group=\'first\' class=\'group\'>')
             .append($container).appendTo($sortable);
 
-        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'second\' class=\'group\' style=\'height: 150px\'><div class=\'test-container\'></div></div>')
+        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'second\' class=\'group\'><div class=\'test-container\'></div></div>')
             .insertAfter($sortable);
 
+        $('#second-group').css('height', '150px');
```

Comment:
Let's add it to the ```<style>```:

```suggestion
        #sortable2 #second-group {
          height: 150px;
        }
```